python/pyspark/mllib/util.py

```diff
@@ -169,6 +169,14 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
```

Comment:
Since the Scala version only operates on individual vectors, this one should not be a wrapper; it should do everything in Python.  The reason is that callMLlibFunc requires the SparkContext and needs to operate on the driver.  But since appendBias operates per-Row, it needs to be called on workers.

Also, please add doc.  Feel free to copy from Scala doc.